src/api/integrations/channel/meta/whatsapp.business.service.ts

```
@@ -395,13 +502,72 @@ export class BusinessStartupService extends ChannelStartupService {
+                  where: {
+                    instanceId: this.instanceId,
+                  },
+                  include: {
+                    OpenaiCreds: true,
+                  },
+                });
+
+                if (
+                  openAiDefaultSettings &&
+                  openAiDefaultSettings.openaiCredsId &&
+                  openAiDefaultSettings.speechToText
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)